lib/tests/teams.nix

```diff
@@ -31,6 +31,10 @@ let
           type = types.str;
           default = "";
         };
+        githubMaintainers = lib.mkOption {
+          type = types.listOf (types.submodule (import ./maintainer-module.nix { inherit lib; }));
```

Comment:
I assume this is unrelated to this PR, but why does `maintainer-module.nix` need to have `{ inherit lib; }` applied? All modules can access `lib` via module args.